backends/graphics/sdl/sdl-graphics.cpp

```diff
@@ -455,7 +472,6 @@ bool SdlGraphicsManager::notifyEvent(const Common::Event &event) {
 		return false;
 	}
 }
-
```

Comment:
Same thing. Please review all files and revert all of these whitespace changes. I will not comment on them more, just please revert.